testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2016-2023 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@buildhost
+Feature: Bootstrap a build host via the GUI
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
```

Comment:
It does not seems we need admin rights here

Reply:
We have to login with some user, or? Which other user could I use?

Reply:
Suggested below.